scripts/publish-action.mjs

```diff
@@ -1,38 +1,46 @@
 #!/usr/bin/env node
 
 import cpy from 'cpy';
-import { execaCommand } from 'execa';
+import { $ } from 'execa';
 import tmp from 'tmp-promise';
 
-const command = (cmd, opts) => execaCommand(cmd, { stdio: 'inherit', ...opts });
-
-const publishAction = async ({ version, repo }) => {
+const publishAction = async ({ major, version, repo }) => {
   const dryRun = process.argv.includes('--dry-run');
 
-  console.info(`✅ Publishing ${version} to ${repo} ${dryRun ? '(dry run)' : ''}`);
+  console.info(`🚀 Publishing ${version} to ${repo} ${dryRun ? '(dry run)' : ''}`);
 
   const { path, cleanup } = await tmp.dir({ unsafeCleanup: true, prefix: `chromatic-action-` });
-  const run = (cmd) => command(cmd, { cwd: path });
 
-  await cpy(['action/*.js', 'action/*.json', 'action.yml', 'package.json'], path, {
-    parents: true,
-  });
-  await cpy(['action-src/CHANGELOG.md', 'action-src/LICENSE', 'action-src/README.md'], path);
+  const copy = (globs, opts) => {
+    console.info(`📦 Copying:\n   - ${globs.join('\n   - ')}`);
+    return cpy(globs, path, opts);
+  };
+
+  await copy(['action/*.js', 'action/*.json', 'action.yml', 'package.json'], { parents: true });
+  await copy(['action-src/CHANGELOG.md', 'action-src/LICENSE', 'action-src/README.md']);
```

Comment:
Is there a reason the path parameter was moved? This reads kind of weird without it. Can we pass it through rather than hiding it in `copy`?